backend/matching-service/socketapp.py

```diff
@@ -46,6 +46,14 @@ async def fetch_questions(difficulty: str):
   questions = await requests.get(f"{os.getenv('QUESTION_SERVICE_URL')}/questions/{difficulty}")
   return questions
 
+async def fetch_questions_random(difficulty: str, email1: str, email2: str):
+    payload = {
+        "email1": email1,
+        "email2": email2
+    }
+    questions = await requests.get(f"{os.getenv('QUESTION_SERVICE_URL')}/questions/{difficulty}/random", json=payload)
+    return questions
+
```

Comment:
Revert to whatever it was before. Just simply get multiple questions (no need to update history) without any payload (call the old API - no need to touch the new one). Later we can change once we decide whether we are going to update history from question service or the frontend